src/AppEntry.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './utilities/store';
+import App from './App';
+import { getBaseName } from '@redhat-cloud-services/frontend-components-utilities/helpers/helpers';
```

Comment:
We should not use second level import paths, that breaks automatic cjs/ems module resolution

```diff
-import { getBaseName } from '@redhat-cloud-services/frontend-components-utilities/helpers/helpers';
+import { getBaseName } from '@redhat-cloud-services/frontend-components-utilities/helpers';
```